source/api/presentation/3.0/index.md

```diff
@@ -47,38 +47,38 @@ This is a work in progress and may change without any notices. Implementers shou
 
 ##  1. Introduction
 
-Access to image-based resources is fundamental to many research disciplines, scholarship and the transmission of cultural knowledge. Digital images are a container for much of the information content in the Web-based delivery of digitized cultural heritage objects. Collections of born-digital images can also benefit from a standardized method to structure their layout and presentation, such as slideshows, image carousels, web comics, and more.
+Access to digital representations of structured resources is a fundamental requirement for many research pursuits, the transmission of cultural knowledge and for the daily pursuits of every web citizen. Digital content is the primary mode of transmission for access to cultural heritage, science and entertainment. Collections of both digitized physical objects and born-digital content can benefit from a standardized description of their structure, layout and presentation mode.
```

Comment:
repetition of _pursuits_: how about "many research activities"?
"can benefit" -> just "benefit" (stronger)